README.md

```diff
@@ -174,41 +174,34 @@ The demo app fetches the plans, protocols and model weights from pygrid server h
 
 Follow these steps to setup an environment to run the demo app:
 
-- Clone the repo [PyGrid](https://github.com/OpenMined/PyGrid) and change directory to it
+- Clone the repo [PyGrid](https://github.com/OpenMined/PyGrid) and change directory to it. At the moment PyGrid doesn't have official releases so please use this [commit](0e93aa645a63a02f45ae72b4ff3106c6402dbadf)  
 
 ```bash
 git clone https://github.com/OpenMined/PyGrid
 cd PyGrid
+git checkout 0e93aa645a63a02f45ae72b4ff3106c6402dbadf
```

Comment:
If there is a tag for this, it would be great to add it here.

Reply:
there is no tag on pysyft. I just used the latest commit but would need to specify the commits for others